### PR TITLE
Add jobservice metrics information

### PR DESCRIPTION
**Pull request by a-mccarthy**
Signed-off-by: Abigail McCarthy <mabigail@vmware.com>

Fixes: https://github.com/goharbor/harbor/issues/14468

Pulling information from the community proposal, https://github.com/goharbor/community/blob/4aea46513c1401589518e9701c6402c9dc275527/proposals/new/metrics_in_jobservice.md. 

Need to verify with a running instance, once this feature is complete. Dev work happing as part of, https://github.com/goharbor/harbor/issues/14469

### CODE REVIEW

**Comment by a-mccarthy** [comment]
To do: 
1. verify the metrics on a live environment
2. fill in the `Labels (Values)` column for the new metrics

**Review by ninjadq — APPROVED** [review]
lgtm

**Comment by a-mccarthy** [comment]
@ninjadq I question about the metrics for jobservice - will reporting happen on all the metrics, even if no jobs are happening? I'm testing this out, but i'm only seeing `harbor_jobservice_info` reported by the jobservice component. Based on the proposal, there should be `harbor_jobservice_task_total` and `harbor_jobservice_task_process_time`, right?

**Review by ninjadq — APPROVED** [review]
lgtm

File: docs/administration/metrics/_index.md [hunk 1]
```diff
@@ -59,6 +63,21 @@ Name | Description | Labels (Values) |Metric type
 `registry_http_request_size_bytes` | The HTTP request sizes in bytes. | handler, le | histogram
 {{< /table >}}
 
+## Harbor Jobservice metrics
+
+The following are metrics pulled from the Docker distribution and are available at `<harbor_instance>:<metrics_port>/<metrics_path>?comp=jobservice`.
```
Comment (ninjadq) [inline docs/administration/metrics/_index.md:68]: these metrics are pulled from the harbor's jobservice component, not Docker distribution

**Review by ninjadq — CHANGES_REQUESTED** [review]

File: docs/administration/metrics/_index.md (hunk 1 above)
Comment (a-mccarthy) [reply docs/administration/metrics/_index.md]: thanks @ninjadq! this is updated.

**Review by ninjadq — APPROVED** [review]

**Pull request merged**